### PR TITLE
Add some abbreviations

### PR DESCRIPTION
**Pull request by stevencychuang**
Add with:
'arr' for 'array'
'def' for 'define'
'dir' for 'directory'
'doc' for 'document'
'fig' for 'figure'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Of course you can judge the not recommended abbreviations. For example why don't
 * **alloc** allocate
 * **alt** alternative/alternate
 * **arg** argument
+* **arr** array
```
Comment (kisvegabor) [inline README.md:41]: I feel it's not shorter enough and I'd rather say it's a not recommended abbreviation. 
Although these things are personal. 

What do you think?

**Review by kisvegabor — COMMENTED** [review]
Thank you, I like them! Only  "arr"/"array" is a little bit small difference for me.

File: README.md (hunk 1 above)
Comment (stevencychuang) [reply README.md:41]: Hi,

Indeed, `arr` might be a kind of personal abbreviation although it is used in my codes frequently.  
Or I could revert the commit, and push again after removing `arr`.   
Thanks for your reply.  

Cheers,  
Steven

**Comment by stevencychuang** [comment]
@kisvegabor Hi,

I have removed "arr" from the branch.  
Feel free to leave your comment if you have any suggestion, thank you.

**Comment by kisvegabor** [comment]
Hi,

Sorry for the late reply. 

Looks good to me, thank you!

**Pull request merged**